rundeckapp/grails-app/views/layouts/base.gsp

```diff
@@ -77,7 +77,9 @@
     <asset:stylesheet href="static/css/components/motd.css"/>
     <asset:stylesheet href="static/css/components/version.css"/>
     <asset:stylesheet href="static/css/components/tour.css"/>
-    <g:if test="${grailsApplication.config.rundeck.communityNews.disabled.isEmpty() ||!grailsApplication.config.rundeck.communityNews.disabled in [false,'false']}">
+    <g:set var="communityNews" value="${cfg.getBoolean(config: 'communityNews.disabled', default: true)}"/>
+
+    <g:if test="${communityNews}">
```

Comment:
I think this logic is wrong (maybe the original logic was wrong too).  the config key  is `.disabled`, so i think the intention is that when unset the communityNews feature is enabled.  only when the config value is set to true will it be disabled.  suggest:
```suggestion
    <g:set var="communityNewsDisabled" value="${cfg.getBoolean(config: 'communityNews.disabled', default: false)}"/>

    <g:if test="${!communityNewsDisabled}">
```